srcpkgs/cinnamon/template

```diff
@@ -1,31 +1,30 @@
 # Template file for 'cinnamon'
 pkgname=cinnamon
-version=5.8.4
-revision=2
+version=6.2.9
+revision=1
 build_style=meson
 build_helper="gir"
 hostmakedepends="pkg-config gettext gettext-devel glib-devel intltool xmlto
  gtk-doc"
 makedepends="cjs-devel clutter-gtk-devel cinnamon-menus-devel gstreamer1-devel
  muffin-devel libgnomekbd-devel polkit-devel libcanberra-devel gtk+3-devel
  startup-notification-devel pulseaudio-devel dbus-glib-devel xapps-devel
- NetworkManager-devel libcroco-devel cinnamon-desktop-devel"
-depends="accountsservice libcaribou cinnamon-settings-daemon>=${version%.*}
- cinnamon-session>=${version%.*} muffin>=${version%.*} gsound
- cinnamon-control-center>=${version%.*} cinnamon-screensaver>=${version%.*}
- cinnamon-translations>=${version%.*} nemo>=${version%.*}
- gnome-themes-standard gnome-themes-standard-metacity gnome-backgrounds
- network-manager-applet polkit-gnome upower>=0.99.7 libkeybinder3 python3-dbus
- python3-gobject python3-pam python3-pexpect python3-Pillow python3-inotify
- python3-tinycss python3-pytz python3-distro python3-requests libtimezonemap
- libsoup"
+ NetworkManager-devel libcroco-devel cinnamon-desktop-devel librsvg-devel"
+depends="accountsservice libcaribou cinnamon-settings-daemon
```

Comment:
restore the version restrictions on the dependencies, this is to prevent broken partial upgrades